src/icons/Status.js

```diff
@@ -12,27 +14,15 @@ export function Valid({ className }) {
     return (
         <svg
             xmlns="http://www.w3.org/2000/svg"
-            width="48"
-            height="48"
             viewBox="0 0 48 48"
             className={className}
         >
             <path d="M24 4C12.95 4 4 12.95 4 24c0 11.04 8.95 20 20 20 11.04 0 20-8.96 20-20 0-11.05-8.96-20-20-20zm-4 30L10 24l2.83-2.83L20 28.34l15.17-15.17L38 16 20 34z" />
-            <style jsx>{`
-                svg {
```

Comment:
Status icons no longer have the correct colour:
![image](https://user-images.githubusercontent.com/185449/63796723-dd390880-c906-11e9-9ed1-408686c12da1.png)

Also true for the other statuses.

Reply:
Aha... yes I removed them too... a little too much cleaning up, sorry. I'm going to fix it in the same way as it was. 

However, in `./src/icons/style.js` I've applied the rule `fill: inherit;`. This is quite handy as it enables us to apply colour to the SVG from a parent element very easily...

For example, this produces a box with a green border and green SVG.
```jsx
const Demo = () => (
    <div>
        <ArrowUp />
        <style jsx>{`
            div {
                border: 1px solid green;
                padding: 24px;
                fill: green;
            }
        `}</style>
    </div>
)
```
So... if we like this type of approach, perhaps we could utilise it here too. IMO it's a more flexible approach then hard-coding the styles into the SVGs.